### ISSUE TITLE
misformatted flow output on MS-Windows

### ISSUE THREAD

**Issue by ikappaki** [jpmonettas/flow-storm-debugger#44]
Hi, 

the output code displayed in the thread tab appears to be confused by windows line endings (`\r\n` compared to unix's `n\`).

![image](https://user-images.githubusercontent.com/34983288/193090230-1c5078b3-85c7-4137-9eb1-6cdd064c5bb4.png)

Way to reproduce (as per getting started guide):

1.  Start a REPL on a PowerShell prompt with flow-storm deps: 
``` powershell
clj -Sforce -Sdeps '{:deps {com.github.jpmonettas/flow-storm-dbg {:mvn/version "RELEASE"} com.github.jpmonettas/flow-storm-inst {:mvn/version "RELEASE"}}}'`
# Downloading: com/github/jpmonettas/flow-storm-inst/maven-metadata.xml from clojars
# Downloading: com/github/jpmonettas/flow-storm-inst/3.0.208/flow-storm-inst-3.0.208.pom from clojars
# Downloading: com/github/jpmonettas/flow-storm-inst/3.0.208/flow-storm-inst-3.0.208.jar from clojars
# Clojure 1.11.1
# user=>
```
2. do a local connect
``` clojure 
Clojure 1.11.1
user=> (require '[flow-storm.api :as fs-api])
nil
user=> (fs-api/local-connect)
[Starting Events queue subsystem]
Events queue started
[Starting Runtime Api subsystem]
[Starting UI subsystem]
;; ...
```
3. In the Browser tab, search for `clojure.string`, right click on it below, and  do `instrument name space: full`. You should see something like below in the repl:
```
user=>
Instrumenting namespace: clojure.string Forms (27) (file:/C:/Users/ikappaki/.m2/repository/org/clojure/clojure/1.11.1/clojure-1.11.1.jar!/clojure/string.clj)
..IIIIIIIIIIIIIIIIIIIIIIIII
```
5. Eval an sform on the REPL
``` clojure
user=> (clojure.string/join " " ["1" "2"])
;; => "1 2"
```
7. Go back to the GUI, the code is formatted as shown above in the screenshot.

I would have had a look myself, but I don't know how to setup a dev env to debug the debugger :) It'd be nice to write to put a small section up on how other people can contribute to the codebase.

Thanks,

**Comment by jpmonettas** [comment]
Hey thanks for reporting @ikappaki ! I was able to reproduce it on a VM.

You are right, I should add instructions on how to debug the debugger since the latest version can't instrument itself.

**Comment by jpmonettas** [comment]
Fixed on https://github.com/jpmonettas/flow-storm-debugger/commit/014bac96ad24bcde4536bbaca6c2163da6cb8152
Released on 2.0.216

Just tried on a windows VM and everything looks fine.

**Comment by ikappaki** [comment]
thanks @jpmonettas , I can confirm it works!
